client/blocks/video-editor/index.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import DetailPreviewVideo from 'post-editor/media-modal/detail/detail-preview-video';
+import VideoEditorButtons from './video-editor-buttons';
+import {
+	videoEditorHasScriptLoadError,
+} from 'state/ui/editor/video-editor/selectors';
+
+class VideoEditor extends Component {
+	static propTypes = {
+		className: PropTypes.string,
+		media: PropTypes.object.isRequired,
+		onCancel: PropTypes.func,
+
+		// Connected props
+		hasScriptLoadError: PropTypes.bool,
+	};
+
+	static defaultProps = {
+		onCancel: noop,
+	};
+
+	handleSelectFrame() {}
+
+	handleUploadImage() {}
+
+	renderError() {
+		const {
+			onCancel,
+			translate,
+		} = this.props;
+
+		return (
+			<Notice
+				status="is-error"
+				showDismiss={ true }
+				text={ translate( 'We are unable to edit this video.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'We are unable to edit this image.' )` *ES Score: 8.31*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).